index.html

```diff
@@ -901,6 +901,16 @@ <h4><code>Timing-Allow-Origin</code> Response Header</h4>
 <li>Return <code>fail</code>.</li>
 </ol>
 </section>
+<section id="sec-iana-considerations">
+<h4>IANA Considerations</h4>
+<p>This section registers <a>Timing-Allow-Origin</a> as a <a href="https://tools.ietf.org/html/rfc3864#section-4.2.2">Provisional Message Header</a>.</p>
+<dl>
+<dt>Header field name:</dt><dd><pre class="abnf">Timing-Allow-Origin</pre></dd>
+<dt>Applicable protocol:</dt><dd><a href="https://tools.ietf.org/html/rfc2616">http</a></dd>
```

Comment:
Should we be pointing at 2616? @mnot /@reschke?

Reply:
Nope.

Reply:
Ok, I've removed the pointer.